libs/inngest/src/integrations/google/api/sheets.ts

```diff
@@ -60,5 +60,58 @@ export async function appendValue({
 		};
 	}
 
-	return `https://docs.google.com/spreadsheets/d/${config.sheet.id}/edit`;
+	return `https://docs.google.com/spreadsheets/d/${config.sheets.filming.id}/edit`;
+}
+
+export async function appendValueForHackathon({
+	email,
+	fullName,
+	githubRepo,
+	deployedApp,
+	tocAgreement,
+	doNotShare,
+}: {
+	email: string;
+	fullName: string;
+	githubRepo: string;
+	deployedApp: string;
+	tocAgreement: boolean;
+	doNotShare: boolean;
+}) {
+	const accessToken = await getGoogleAccessToken();
```

Comment:
TODO might want to move this into its own step in the workflow in case we do other Google things and want to share the token